doc/API/Bills.md

```diff
@@ -176,4 +236,42 @@ Output:
  ],
  "count": 1,
 }
-```
\ No newline at end of file
+```
+
+### `get_bill_history_graph`
+
+Retrieve a graph of a previous period of a bill
+
+Route: `/api/v0/bills/:id/history/:bill_hist_id/graphs/:graph_type`
+
+Input:
+
+Example:
+
+```curl
+curl -H 'X-Auth-Token: YOURAPITOKENHERE' https://librenms.org/api/v0/bills/1/history/1/graphs/bits
+curl -H 'X-Auth-Token: YOURAPITOKENHERE' https://librenms.org/api/v0/bills/1/history/1/graphs/hour
+curl -H 'X-Auth-Token: YOURAPITOKENHERE' https://librenms.org/api/v0/bills/1/history/1/graphs/day
+```
+
+Output:
+
+(image)
+
+### `get_bill_history_graphdata`
+
+Retrieve the data for a graph of a previous period of a bill, to be rendered in an external system
+
+Route: `/api/v0/bills/:id/history/:bill_hist_id/graphdata/:graph_type`
+
+Input:
+
+Example:
+
+```curl
+curl -H 'X-Auth-Token: YOURAPITOKENHERE' https://librenms.org/api/v0/bills/1/history/1/graphdata/bits
+curl -H 'X-Auth-Token: YOURAPITOKENHERE' https://librenms.org/api/v0/bills/1/history/1/graphdata/hour
+curl -H 'X-Auth-Token: YOURAPITOKENHERE' https://librenms.org/api/v0/bills/1/history/1/graphdata/day
```

Comment:
The output for /day:

```
            "17\nJan",
            "18\nJan",
            "19\nJan",
            "20\nJan",
```

Should we add `\n` here as that should be potentially dealt with on the client end. Instead do `17 Jan` or `17-Jan`

Reply:
Just trying to work out the best way to handle this - it's applicable to all of the transfer graphs.  Wondering whether to just return the unix timestamps of the start/end of each item and let all formatting be done on the client end.  Only potential problem is if it introduces a timezone dependency